fala/common/mixin_for_views.py

```diff
@@ -23,7 +24,8 @@ def get_context_data(self, **kwargs):
 
 class CategoryMixin:
     def setup_category(self, request, *args, **kwargs):
-        category_code = request.GET.get("categories")
+        category_code = request.GET.get("categories", "")
+        sub_category_code = request.GET.get("sub-categories", "")
```

Comment:
query? will this work if multiple sub-categories are provided? If not we should rename this to sub-category as i think we are only ever expecting to be provided with one secondary `sub-category` so this would be consistent with that.

Reply:
@willc-work - Yeah that makes sense, happy to change the nomenclature to `sub-category`.

Currently it's set-up for one sub-category.